CompetitionTemplate/src/main/java/competition/subsystems/elevator/ElevatorSubsystem.java

```diff
@@ -0,0 +1,60 @@
+package competition.subsystems.elevator;
+
+import com.google.inject.Inject;
+import com.google.inject.Singleton;
+
+import xbot.common.command.BaseSubsystem;
+
+
+@Singleton
+public class ElevatorSubsystem extends BaseSubsystem {
+	
+	@Inject
+	public ElevatorSubsystem() {
+		// TODO Auto-generated constructor stub
+	
+	}
+
+	/**
+	 * the elevator will rise by certain speed, the speed may change
+	 */
+	public void rise(){
+		
+	}
+	
+	
+	/**
+	 * the elevator will fail by certain speed, the speed may change
+	 */
+	public void lower(){
+	//move when hold the button??
```

Comment:
Remove this comment - the "why" e.g. a button triggering this is not important.

Reply:
Okay.